packages/warehouses/src/warehouseClients/PostgresWarehouseClient.ts

```diff
@@ -252,16 +261,36 @@ export class PostgresClient<
     }
 }
 
+// Mimics behaviour in https://github.com/brianc/node-postgres/blob/master/packages/pg-connection-string/index.js
+const getSSLConfigFromMode = (mode: string): PoolConfig['ssl'] => {
+    switch (mode) {
+        case 'disable':
+            return false;
+        case 'prefer':
+        case 'require':
+        case 'allow':
+        case 'verify-ca':
+        case 'verify-full':
+            return {
+                ca: POSTGRES_CA_BUNDLES,
```

Comment:
Do we need to check the host value like we do in the profiles.ts ? 
```credentials.host.endsWith('.rds.amazonaws.com')```
Will this have an impact on all the postgres connections that aren't hosted in aws ?

Reply:
So node doesn't use postgres drivers at all to communicate with postgres and uses the node TLS package to communicate. So it's quite different to how it works with `profiles.ts` 

With node we can actually specific a whole array of certificates. All I've done is append the RDS one to the list of built in certs (which is the mozilla CA cert bundle).

So this should include _all_ certs we already had PLUS the RDS ones. With the postres driver (used by dbt) we have to point to a single file of certificates

BUT you're right, this code will touch all postgres deployments (I don't think there are many of those in production at all). If we added the if statement, we would reduce the amount of users affected by this change. But also might be more confusing for devs, since it's not really necessary.